packages/openactive-integration-tests/test/features/core/availability-check/implemented/availability-confirmed-test.js

```diff
@@ -1,16 +1,11 @@
 /* eslint-disable no-unused-vars */
 const chakram = require('chakram');
 const chai = require('chai'); // The latest version for new features than chakram includes
-const { RequestState } = require('../../../../helpers/request-state');
```

Comment:
These dependencies are unused in the test. Lint didn't capture this 🤔 . Problem for another PR perhaps

Reply:
/* eslint-disable no-unused-vars */ disables the check for them being unused. The idea here was just to have a bunch of boilerplate dependencies included in each test... though these could actually be consolidated into a helper / index which is probably the better approach here?

Reply:
Hah! I think I stumbled over the triple negative "disable no unused vars"

Reply:
@nickevansuk I personally agree with the ESLint rule for this. If we allow extraneous dependencies and no way to check, the code could end up getting messy.

Consolidation of dependencies into an index seems like an unnecessary layer of indirection to me. (One thing that I think is good to optimise is how many files a developer has to click through in order to look into something).

With VSCode (aided by TS), it should be possible to just type `RequestState` and the import automatically be added to the top of the file

Reply:
Sure sounds good to me!